README.md

```diff
@@ -1,7 +1,21 @@
-# VoiceCollector
+## Welcome to Voice Collector! 🎙️
 
+Hello there! We're thrilled to have you here. Our application is not just any ordinary data collection platform. We're on a mission to make voice recognition inclusive for everyone, especially for those with speech impairments.
 
-## Dev Guide
-Create your *own branch* to do implemetation.
-After implementation, push the changes to your branch then create a pull request to dev branch.
-DO NOT PUSH TO MAIN!
+### **Why is Our Dataset Special? 🌟**
+
+1. **A Heart for Impaired Speakers**: Our platform stands out because it focuses on impaired speakers, making sure their voices are heard and recognized.
+
+2. **A Treasure Trove of Data**: We're proud to offer a dataset packed with high-quality audio from dysarthria speakers, detailing different severity levels. As of now, this is the most extensive public dataset in this unique domain.
+
+3. **Powered by Cutting-Edge Tech**: We've embraced wav2vec 2.0, a top-notch pre-trained speech recognition model, to refine our validation process. This ensures pristine data quality and also provides additional insights like intelligibility scores.
```

Comment:
I removed wav2vec2.0 on this @ztybigcat

Reply:
We do not do ASR validation on it, the models were too heavy.

Reply:
I would write a bit more about the layout of our project, sounds dry but what our front-end and backend are in and so on.

Reply:
Replaced those content with an overview of different technologies used in the application.